yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -604,10 +617,19 @@ JSON_TABLE(data, '$.skills[*]'
 			NewJsonQueryFunctionIssue(DML_QUERY_OBJECT_TYPE, "", sqls[12], []string{JSON_VALUE, JSON_EXISTS}),
 		},
 		sqls[13]: []QueryIssue{
-			NewJsonQueryFunctionIssue("MVIEW", "public.test_jsonb_view", sqls[13], []string{JSON_VALUE, JSON_EXISTS, JSON_TABLE}),
+			NewJsonPredicateIssue(DML_QUERY_OBJECT_TYPE, "", sqls[13]),
 		},
 		sqls[14]: []QueryIssue{
-			NewJsonConstructorFunctionIssue(DML_QUERY_OBJECT_TYPE, "", sqls[14], []string{JSON_ARRAY}),
+			NewJsonPredicateIssue(DML_QUERY_OBJECT_TYPE, "", sqls[14]),
+		},
+		sqls[15]: []QueryIssue{
+			NewJsonPredicateIssue(DML_QUERY_OBJECT_TYPE, "", sqls[15]),
+		},
```

Comment:
In general, lets try to keep the unit tests as redudant as possible moving forward. Unless something new is being tested we should not write duplicate tests.

I think here its enough to just have one test case of json predicate issue. 
We don't need to test the pg_query module if its detecting all expected ones as json predicate or not.
Checking some of these in a single should be enough.

Reply:
Checking in a single query won't suffice because we will return the issue on that query if we find a single type of it. So it's good to test all the different types.